components/community/get-involved/GetInvolved.js

```diff
@@ -71,19 +71,12 @@ export default function GetInvolved() {
             </h4>
             <p>Want to find out the active proposals proposed by the Ballerina community? Check out the <a className={styles.getStartLinks} rel="noreferrer" target="_blank" href="/community/active-proposals">active proposals list</a>.</p>
           </Col>
-          <Col xs={12} md={4} lg={4} className={styles.card}>
-            <span className="bookMarkOnPage" id="submit-a-use-case"></span>
-            <h4 id="submit-a-use-case-title">
-              Submit a use case
-            </h4>
-            <p><a id="submitUseCase" className={styles.getStartLinks} onClick={handleShow}>Tell us</a> how you&apos;re using Ballerina in your project and be a part of our community stories. </p>
-          </Col>
           <Col xs={12} md={4} lg={4} className={styles.card}>
             <span className="bookMarkOnPage" id="host-a-ballerina-event"></span>
             <h4 id="host-a-ballerina-event-title">
               Host a Ballerina event
             </h4>
-            <p>Want to talk about Ballerina at your local tech meetup? Reach us at <a className={styles.getStartLinks} href="mailto:contact@ballerina.io">contact@ballerina.io</a>, and we will help you with any presentation content.</p>
+            <p>Want to talk about Ballerina at your local tech meetup? Reach us at <a className={styles.getStartLinks} href="mailto:contact@ballerina.io">contact@ballerina.io</a>.</p>
           </Col>
           
 
```

Comment:
You can remove the model component since we're not using it any more in L83-L90.

Reply:
Removed.